web/src/billing/helpers.ts

```diff
@@ -39,11 +57,11 @@ export function create_ajax_request(
         data[item.name] = item.value;
     }
 
-    $.ajax({
+    void $.ajax({
         type,
         url,
         data,
-        success(response) {
+        success(response: T) {
```

Comment:
This `response` parameter comes from a value of type `any`, so its validity as type `T` is completely unchecked. We should write `response: unknown`, and the caller should use [Zod](https://zod.dev/) to safely parse and validate it as the desired type.

Reply:
@andersk Thanks for the review I have integrated Zod but how do I fix failing tests now should I make every field nullable in my Zod schema? I would like your opinions on this!